### DISCUSSION TITLE
インストール中に問題が発生しましたI encountered a problem while installing

### DISCUSSION THREAD

**Discussion by wzdret** [ngmansion/FE8N#101] [Q&A]
BPSファイルは正常に機能しますが、ソースコードを使用してEAでインストールすると、ゲームがクラッシュします。                                                The BPS file can work normally, but the game will crash when using the source code to install it with EA.

**Comment by aera-ngmansion** [comment]
~~こんにちは。~~
~~恐らくですが、FE8Nの構成に関して誤解があると思われるので、そのご説明させていただきます。~~

~~FE8Nは、EAパッチ単独では完成しておりません。~~
~~bpsパッチをメインに据えており、そのサブ（アップデート用）としてEAパッチを提供しております。~~

~~従って、クリーンROMに直接EAでパッチしても動作しません。~~
~~bpsパッチを当てた後にEAパッチを当てていただくようお願いします。~~

コミット忘れでレポジトリの内容と bpsパッチの内容が乖離している事によるバグでした。
直します。

**Comment by aera-ngmansion** [comment]
a953d89
こちらのコミットで直せたと思います。
